puffin-imgui/src/ui.rs

```diff
@@ -356,8 +357,9 @@ impl ProfilerUi {
 
             self.options.pixels_per_ns *= zoom_factor;
             let zoom_center = painter.mouse_pos.x - painter.canvas_min.x;
-            self.options.sideways_pan_in_pixels =
-                (self.options.sideways_pan_in_pixels - zoom_center) * zoom_factor + zoom_center;
+            self.options.sideways_pan_in_pixels = (self.options.sideways_pan_in_pixels
+                - zoom_center)
+                .mul_add(zoom_factor, zoom_center);
```

Comment:
Fewer parenthesis (again):

```suggestion
            self.options.sideways_pan_in_pixels = zoom_factor.mul_add(
                    self.options.sideways_pan_in_pixels - zoom_center, zoom_center);
```